saplings/profile/src/Profile.js

```diff
@@ -83,7 +83,20 @@ export function Profile() {
     setModalActive(true);
   };
 
+  const sortKeysActive = (allKeys, publicKey) => {
+    const activeKey = allKeys.find(key => key.public_key === publicKey);
+    const keyIndex = allKeys.findIndex(key => key.public_key === publicKey);
```

Comment:
Looks like you could just do this search, and then directly index for the activeKey:
```javascript
const keyIndex // same as this line
const activeKey = allKeys[keyIndex];
```

Reply:
Fixed